utils/square_fraction.py

```diff
@@ -0,0 +1,6 @@
+# Checks if the square result has fraction and throws error.
+def square_fraction(user_num, square_result):
+    if (square_result).is_integer():
+        print(f"The square of {int(user_num)} = {int(square_result)}")
+    else:
+        print ('Error: Your square result is not an integer!')
```

Comment:
The error message is not clear enough, change the "Your square result" to "The outcome of your square calculation is not a whole number."